app/src/main/java/org/simple/clinic/widgets/PatientSearchResultItemView.kt

```diff
@@ -93,6 +107,48 @@ class PatientSearchResultItemView(
     renderPatientPhoneNumber(searchQuery, model)
     renderVisited(model.lastSeen)
     renderLastSeen(model.lastSeen, currentFacilityId)
+    renderIdentifier(model.identifier, searchQuery)
+  }
+
+  private fun getId(identifier: Identifier, searchQuery: String): Id {
+    val id = when (identifier.type) {
+      BpPassport -> BpPassport.shortCode(identifier)
+      BangladeshNationalId -> identifier.value
+      EthiopiaMedicalRecordNumber -> identifier.value
+      IndiaNationalHealthId -> identifier.value
+      is Unknown -> throw IllegalArgumentException("Unknown identifier: $identifier")
```

Comment:
It seems like whenever we display a `BP Passport` anywhere in the UI, we always display the short code and not the full value. In this case, do you think it's worth extract this `when` clause into the `Identifier` type as a computed property, say `displayText` or something?

Reply:
Yep, we already have a `displayValue` that does format the search results. But it adds specific Unicode characters depending on the identifier type. I felt like this display variant is only used here, so that's why I didn't move it to the `Identifier` class. Should I move it?

Reply:
Ah, that makes sense. Let's leave this as is for now then.